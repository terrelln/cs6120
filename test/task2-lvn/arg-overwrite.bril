# ARGS: 4 2
@main(a: int, b: int) {
    c: int = id a;
    d: int = id b;
    e: int = add a b;
    f: int = add a b;
    g: int = id e;
    a: int = id e;
    b: int = id f;
    h: int = add a b;
    i: int = add c d;
    j: int = add c d;
    k: int = id a;
    l: int = id b;
    print c;
    print d;
    print e;
    print f;
    print g;
    print h;
    print i;
    print j;
    print k;
    print l;
}