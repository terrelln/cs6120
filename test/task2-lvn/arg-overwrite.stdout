4
2
6
6
6
12
6
6
6
6
